src/Tempest/Database/src/Connections/PostgresConnection.php

```diff
@@ -21,6 +23,7 @@ public function __construct(
         public string $password = '',
         #[SensitiveParameter]
         public string $database = 'postgres',
+        public NamingStrategy $namingStrategy = new PluralizedSnakeCaseStrategy()
```

Comment:
Haven't read through the rest of the code, but my first thought is that this property belongs in `DatabaseConfig` and not the connection

Reply:
[This](https://discord.com/channels/1236153076688359495/1236153079603269766/1287067156835602588) might be relevant.

Reply:
Nope, I don't see where it got discussed?

Reply:
Yeah I'm not quite sure yet how multi-database support would work. I think keeping this on database config makes most sense for now :)